standalone/ansible/roles/ssh/files/ssh_keys/production/semaphore-ci

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQCt/h+ObkK2dDhFe07QCrH33D8ER79buKWzd1aH5cZPV73c6ffM4T7PQfC2RLQ+joEg7QxxZ20WKzOUudhtP+2jKeyIJ2pn7FahscrjM7Pd/OjgvZMgeXRHzeWDmwsd23F9yjepi2Gg6WKjjAqLHq4jUB84YBzrVXxAw99rwZwEicOpPMEbT8hJU67FSRHJr0B99+pqp9Zt4v7qqTWaoVOQML2MFVoJfZu7cm9nD/7K0SYsJgyCdALkMns6zhEmbaA8G2Wc928ivdyNDFShletYoq4B53kZPH4jWipQo7SVGo93krJtKDGtUHGgBk2llfF/fPRj7aO4lMVVPCf/CxzI22FuWqofkeIbwOELpPmi5uhB+DchGQhP23cbaEUdfEA/jewpubQZG7+HqWTZbGSLD0f63UhkB+WaIw1XnW8VG4U55zJJ8jWlJZYHsqkCIY+tV0XPpDfyGNaUmWd5+Taboa/cES/AKdwgdBP/VCcbnexOKMiXnIEbXQfk06IQwIGZlR5mYCB5ZYYjuwqLrCIyYYDlNcFh+fLO/Dxkd2uZo3M2WZuuEmiUG3MvDAHIuMnrV4eqgYGbrLKYbhS2UTfe5mv1vPjCmJWk+NE2vVKMw7PKkE6hYSglikR+hplXVHE1Ep5eK1dMjsdmY4QglFzKdpehJsSrggn7u8PMbkp+aQ== semaphore@simple.org
```

Comment:
Does Semaphore need access to production? Do we expect Semaphore to push config or code changes to prod directly?

Reply:
I'm not sure why this key was added, my guess is this was copied over from the other ansible which sets up `SBX` and `QA`. I think we can get rid of it.

Reply:
For reference, let's check if our IHCI and BD prod envs include it.

Reply:
They do. I'm not sure if they should be included?

Reply:
Let's keep em. We can investigate out of band.